app/src/main/java/org/schabi/newpipe/settings/preferencesearch/PreferenceFuzzySearchFunction.java

```diff
@@ -72,39 +73,23 @@ static class FuzzySearchSpecificDTO {
         );
 
         private final PreferenceSearchItem item;
-        private final float score;
+        private final double score;
```

Comment:
Wait, why did you convert it to `double`?

Reply:
> Wait, why did you convert it to `double`?

The streams API doesn't support mapping to `float` directly. It works as it did before, though.